lib/ansible/modules/cloud/docker/docker_volume.py

```diff
@@ -239,7 +239,7 @@ def main():
         state=dict(type='str', default='present', choices=['present', 'absent']),
         driver=dict(type='str', default='local'),
         driver_options=dict(type='dict', default={}),
-        labels=dict(type='list'),
+        labels=dict(type='dict'),
```

Comment:
Could you please update documentation on line 41 as well ? also a note about this change in [changelog.md](https://github.com/ansible/ansible/blob/devel/CHANGELOG.md#minor-changes)